microverse-trials-leo-joel-main/fcc-test-suite-template-master/css/style.css

```diff
@@ -0,0 +1,19 @@
+   @media (min-width:1025px) {
+      body {
+        background-color: rgb(241, 239, 239);
+      }
+    }
+
+#welcome-section{
+height: 100vh;
+padding-top: 10rem;
+}
+
+body{
+scroll-behavior: smooth;
+}
+
+#navbar{
+position: fixed;
+width: 100%;
+}
```

Comment:
Kindly correct the indentation and spacing of style.css to follow CSS's best practices.